src/python/pants/backend/docker/goals/package_image.py

```diff
@@ -268,22 +273,28 @@ async def build_docker_image(
             process_cleanup=process_cleanup.val,
         )
 
-    logger.debug(
-        dedent(
-            f"""\
-            Docker build output for {tags[0]}:
-            stdout:
-            {result.stdout.decode()}
-
-            stderr:
-            {result.stderr.decode()}
-            """
+    if options.build_verbose:
+        logger.info(
```

Comment:
Maybe we should always log this, but at debug level if the verbose flag is not set and at info level if it is? 

Also, why not differentiate stdout and stderr, like before?

Reply:
> Maybe we should always log this, but at debug level if the verbose flag is not set and at info level if it is?

When the flag is not set, we pass `--quiet` and there is no output to log. (besides the image id, that is captured from stdout later)

> Also, why not differentiate stdout and stderr, like before?

The build output is logged on stderr, and stdout was always empty, so it was to reduce some lines of the what I felt like noise, but if you think it is useful to show that the info was on stderr, we can put it back :)

Reply:
Hmm, right. Actually, no harm in logging the output in debug mode when the flag is not set. That way it shows what it was, or not, in the logs, rather than leaving users wondering. Will fix.